rocketmq-amqp/src/main/java/org/apache/rocketmq/amqp/EndpointIdentifier.java

```diff
@@ -0,0 +1,40 @@
+package org.apache.rocketmq.amqp;
+
+import java.util.Map;
+import org.apache.rocketmq.amqp.message.AMQPReceiver;
+import org.apache.rocketmq.amqp.message.AMQPSender;
+
+public class EndpointIdentifier {
+    public String findEndpoint()
+    {
+        String endpoint ="producer";
+
+
+        return endpoint;
+    }
+    public void execute(Map messageProperties) throws Exception {
+        String endpoint = messageProperties.get("ep").toString();
+        switch (endpoint) {
+            case "producer" :
+              AMQPSender sender = new AMQPSender();
+              sender.producer(messageProperties);
```

Comment:
one object of AMQPSender for one message? In fact, it may need only one Producer.